internal/internal.go

```diff
@@ -308,7 +308,22 @@ func (p *envoyExtAuthzGrpcServer) check(ctx context.Context, req interface{}) (*
 
 	input, err = envoyauth.RequestToInput(req, logEntry, p.cfg.protoSet)
 	if err != nil {
-		return nil, stop, err
+		logrus.WithField("err", err).Info("Unable to parse request.")
+
+		deniedResponse := &ext_authz_v3.DeniedHttpResponse{
+			Status: &ext_type_v3.HttpStatus{
+				Code: ext_type_v3.StatusCode(ext_type_v3.StatusCode_BadRequest),
+			},
+		}
+
+		resp := &ext_authz_v3.CheckResponse{
+			HttpResponse: &ext_authz_v3.CheckResponse_DeniedResponse{
+				DeniedResponse: deniedResponse,
+			},
+			Status: &rpc_status.Status{Code: int32(code.Code_INVALID_ARGUMENT)},
```

Comment:
It just now strikes me as odd that you've got to define both the http status and this one. 🤔 

From the docs, the inner one would overwrite the outer one. So let's only specify one of the two? This would also make it simpler to configure this.